azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -29,13 +49,22 @@ def to_pandas_timedelta(raw_value: Union[int, float, str]) -> "pandas.Timedelta"
             return pd.to_timedelta(formatted_value)
 
 
-def dataframe_from_result_table(table: "Union[KustoResultTable, KustoStreamingResultTable]", nullable_bools: bool = False) -> "pandas.DataFrame":
-    """Converts Kusto tables into pandas DataFrame.
+def dataframe_from_result_table(
+    table: "Union[KustoResultTable, KustoStreamingResultTable]",
+    nullable_bools: bool = False,
+    converters_by_type: dict[str, Callable[[str, "pandas.DataFrame"], "pandas.Series"]] = None,
```

Comment:
the dict[...] syntax is python 3.10+
You need to import Dict from typings and do Dict[...]